code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1124,25 +1168,25 @@
 
 //Skrell 'hairstyles'
 /datum/sprite_accessory/hair/skr_tentacle_veryshort
-	name = "Skrell Short Tentacles"
-	icon_state = "skrell_hair_short"
-	species_allowed = list(SPECIES_SKRELL)
+	name = "Skrell Very Short Tentacles"

```

Comment:
You've reverted the change to Skrell hair here, which is a step back regarding accuracy to the wiki and will also break save files made since then.